engine/apps/api/views/alert_receive_channel.py

```diff
@@ -622,3 +629,50 @@ def disconnect_contact_point(self, request, pk):
         if not disconnected:
             raise BadRequest(detail=error)
         return Response(status=status.HTTP_200_OK)
+
+    @extend_schema(request=None, responses=WebhookSerializer(many=True))
+    @action(detail=True, methods=["get"], url_path="webhooks")
+    def webhooks_get(self, request, pk):
+        instance = self.get_object()
+        return Response(
+            WebhookSerializer(
+                instance.webhooks.filter(is_from_connected_integration=True),
```

Comment:
Wondering if it would make sense to abstract the base queryset in a method so we always start from there to work with webhooks in this context (`get_webhooks_queryset`?)